apps/cb-ceiling/test/cbQueryTests/doc/readme.txt

```diff
@@ -0,0 +1,11 @@
+Configuration is loaded from ./config/config.json
```

Comment:
I believe we're trying to keep in-repo documentation as Markdown files. Would it be possible to reformat this into a `README.md` file?

Reply:
@ian-noaa thanks for the suggestion, moved doc/readme.txt as ./README.md
I am working on the couchbase 4.x upgrade to the app, will move dependency to app level when that looks good.